tests/page-objects/default/enketo/death-report.page.js

```diff
@@ -0,0 +1,66 @@
+const moment = require('moment');
+const genericForm = require('./generic-form.wdio.page');
+
+const PLACE_OF_DEATH = {healthFacility: 'health_facility', home: 'home', other: 'other'};
+
+const FORM = 'form[data-form-id="death_report"]';
+const SUMMARY_SECTION = `${FORM} section[name="/death_report/group_review"]`;
+
+const deathDate = () => $(`${FORM} section[name="/death_report/death_details"] input.ignore.input-small`);
+const deathPlace = (value) => $(`${FORM} input[name="/death_report/death_details/place_of_death"][value="${value}"]`);
+const deathInformation = () => $(`${FORM} textarea[name="/death_report/death_details/death_information"]`);
+const patientNameSummary = () => $(`${SUMMARY_SECTION} ` +
+  `span[data-itext-id="/death_report/group_review/r_patient_details:label"] ` +
+  `span[data-value=" /death_report/patient_display_name "]`);
+const deathDateSummary = () => $(`${SUMMARY_SECTION} span[data-value=" /death_report/death_details/date_of_death "]`);
+const deathInformationSummary = () => $(`${SUMMARY_SECTION} ` +
+  `span[data-value=" /death_report/death_details/death_information "]`);
+
+const setDeathDate = async (value = moment().subtract(1, 'day').format('YYYY-MM-DD')) => {
+  const date = await deathDate();
+  await date.waitForDisplayed();
+  await date.setValue(value);
+};
+
+const selectDeathPlace = async (value = PLACE_OF_DEATH.healthFacility) => {
+  const place = await deathPlace(value);
+  await place.waitForClickable();
+  await place.click();
+};
+
+const setDeathInformation = async (text = 'Test note') => {
+  const note  = await deathInformation();
+  await note.waitForDisplayed();
+  await note.setValue(text);
+};
+
+const getSummaryDetails = async () => {
+  return {
+    patientNameSummary: await patientNameSummary().getText(),
+    deathDateSummary: await deathDateSummary().getText(),
+    deathInformationSummary: await deathInformationSummary().getText(),
+  };
+};
+
+const submitDeathReport = async ({
+  deathDate: deathDateValue = moment().subtract(1, 'day').format('YYYY-MM-DD'),
```

Comment:
What is the benefit of adding the defaults values here again? Clarity 🤔 
For example: `setDeathDate` function already has the default value.

Reply:
There is no benefit at all. Sometimes I forget that in javascript there are a lot of things that you can do that you can't in other languages.
Being completely honest I thought that if didn't put a default value in that function it will fail if I try to call it without parameters 😅 
My mistake, thank you for point it out!